dji_sdk/include/dji_sdk/dji_sdk_node.h

```diff
@@ -54,8 +56,11 @@ class DJISDKNode
     nav_msgs::Odometry odometry;
 	dji_sdk::TimeStamp time_stamp;
     sensor_msgs::Imu imu_msg;
+    sensor_msgs::NavSatFix gps_msg;
 	dji_sdk::A3GPS A3_GPS;
 	dji_sdk::A3RTK A3_RTK;
+// Declare Time translator
+    boost::shared_ptr<cuckoo_time_translator::DeviceTimeUnwrapperAndTranslator> device_time_translator_; // TODO: Why does this have to be a boost::shared_ptr ?
```

Comment:
The `boost::shared_ptr` is not needed. In the example driver this was the simplest available solution (pre c++11) to the deferred initialization requirement that came with the fact that the node handle was it requires was created in the `onInit` method. 
As your class gets the node handle from outside passed to the constructor you don't need a deferred initialization at all and could make this a simple member and move its initialization to the initialization list of the constructor.